Nanoleaf-Models/Models/UserSettings.cs

```diff
@@ -13,6 +13,7 @@ namespace Winleafs.Models.Models
     public class UserSettings
     {
         public static readonly string APPLICATIONNAME = "Winleafs";
+        public static readonly string APPLICATIONVERSION = "v0.1.2";
```

Comment:
We are at version 0.2 now right?

Reply:
0.2 is prerelease so the API doesn't give it back as latest. This is technically our newest non prerelease version.